libs/design-system/styles/_typography.scss

```diff
@@ -69,6 +69,8 @@ $overrides: (
 
   body-large-tracking: 0.0313rem,
 
+  body-large: 400 1rem / 1.5rem $_plain,
```

Comment:
This is already the default for body-large so there is no need to override it. Please undo this change